ambry-vcr/src/test/java/com/github/ambry/vcr/CloudTokenPersistorTest.java

```diff
@@ -169,7 +169,7 @@ public void testPersistToken() throws IOException {
     token =  new StoreFindToken(FindTokenType.Uninitialized, null, null, null, null,
         true, (short) 3, null, null, (short) -1);
     response = new ReplicaThread.ExchangeMetadataResponse(Collections.emptySet(), token,
-            -1, Collections.emptyMap(), new SystemTime());
+        -1, Collections.emptyMap(), new SystemTime());
```

Comment:
can u restore this file from master ? there are no code changes here, just some formatting changes due to different IDE settings..

Reply:
fixed.